sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -936,6 +936,12 @@ public sealed partial class TrueRuleFilter : Azure.Messaging.ServiceBus.Administ
 }
 namespace Microsoft.Extensions.Azure
 {
+    public static partial class ServiceBusAdministrationClientBuilderExtensions
```

Comment:
I would just add these methods to the existing ServiceBusClientBuilderExtensions class.

Reply:
Moved them as per your suggestion, had them seperate since as far as I know ServiceBusAdminClient is not a ServiceBusClient (did not spot inheritance or something like that). But for me its fine either way :)

Reply:
I think the name of the class means that it is for clients used with Service Bus as opposed to ServiceBusClient specifically.

Reply:
Gotcha, makes as much sense as my own interperation :)